api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -140,28 +132,43 @@
     </tr>
     <tr>
       <td colspan="3">
-        <div class="text">Quick guide on how to get started:</div>
+        <div class="text">Helpful tips before you get started:</div>
       </td>
     </tr>
     <tr class="card">
-      <td>
-        <a class="unstyled" href="https://myaccount.google.com/signinoptions/two-step-verification"><img class="bullet" src="${BULLET_1}"></a>
-      </td>
-      <td colspan="2">
-        <a class="unstyled" href="https://myaccount.google.com/signinoptions/two-step-verification"><span class=" card-text link">Sign into Google</span><span> with your new account, change your password, and set up 2-Step Verification</span></a>
+      <td colspan="3">
+        <div style="display: flex">
+          <div class="bullet">
+            <img src="${BULLET_1}">
+          </div>
+          <div class="text">
+            <a class="text-link" href="https://support.google.com/chrome/answer/95346">Chrome</a>
+            <span> is the primary browser we currently support for the Researcher Workbench.
+            We highly recommend using Chrome for the best user experience.
+          </span>
+          </div>
+        </div>
       </td>
     </tr>
+    <tr style="height: 1em"></tr>
     <tr class="card">
-      <td>
-        <a class="unstyled" href="https://myaccount.google.com/intro/recovery/email?hl=en-US"><img class="bullet" src="${BULLET_2}"></a>
-      </td>
-      <td colspan="2">
-        <a class="unstyled" href="https://myaccount.google.com/intro/recovery/email?hl=en-US"><span class="card-text link">Add</span><span> your recovery email address</span></a>
+      <td colspan="3">
+        <div style="display: flex">
+          <div class="bullet">
+            <img src="${BULLET_2}">
+          </div>
+          <div class="text">
+            <span>
+              We recommend creating a <a class="text-link" href="https://support.google.com/chrome/answer/2364824">Chrome Profile </a>
+              with your All of Us Researcher Workbench Google account. This will keep your Workbench browser sessions isolated from your other Google accounts.
```

Comment:
Wording: feels a bit presumptuous to assume that all users have existing Google accounts. Possible changes:

- "This will keep your Workbench browser sessions isolated from your other Google accounts, if any."
- This will keep your Workbench browser sessions isolated if you use other Google accounts."
- "If you have other Google accounts, this will keep your Workbench browser sessions isolated."

Reply:
We are going with "This will keep your Workbench browser sessions isolated if you use other Google accounts."